templates/swagger-ui/swagger.json

```diff
@@ -1086,6 +1071,186 @@
                   }
               ]
           }
-      }
+      },
+      "/capacity/s3/{resource}/{id}": {
+          "get": {
+            "tags": [
+                "Cortx Capacity"
+            ],
+            "summary": "Get user level Capacity usage statistics.",
+            "responses": {
+                "200": {
+                    "description": "OK"
+                },
+                "401": {
+                    "description": "Unauthorized: Token has not found or invalid format or session is invalid"
+                },
+                "403": {
+                    "description": "Forbidden: This error is raised for all cases when user don't have permissions"
+                },
+                "503": {
+                    "description": "Unable to connect to the service"
+                },
+                "500": {
+                    "description": "Internal Server Error"
+                }
+            },
+            "content": {
+                "application/json": {}
+            },
+            "parameters": [
+                {
+                    "name": "resource",
+                    "in": "path",
+                    "description": "The resource whose capacity usage need to check.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    },
+                    "example": "user"
+                },
+                {
+                    "name": "id",
+                    "in": "path",
+                    "description": "Get capacity usage by user id",
```

Comment:
"Get capacity usage by user id" -> id wont be always of user, it can be bucket id too. Please modify summary accordingly

Reply:
updated